recipes/xtl/all/test_package/conanfile.py

```diff
@@ -1,6 +1,6 @@
 import os
 
-from conans import ConanFile, CMake, tools
```

Comment:
Can you, instead of removing `tools` do the following modification?
```
    def test(self):
        if tools.cross_building(self.settings):
            self.run(os.path.join("bin", "test_package"), run_environment=True)
```